tox.ini

```diff
@@ -94,7 +95,7 @@ setenv =
     BLACK_LINT_ARGS=--check
 
 commands =
-    black {env:BLACK_LINT_ARGS:} setup.py src
+    black {env:BLACK_LINT_ARGS:} {posargs:setup.py src}
```

Comment:
This is to allow one to specify specific files.

Reply:
makes sense